repository/models/queries/asset.py

```diff
@@ -72,7 +90,14 @@
 }
 """
 
+# Gets asset by entity id
```

Comment:
'Gets' is maybe not the right word, we don't return the whole asset object but just identifiers to reference the asset.  Idem for next query.